src/main/java/io/antmedia/datastore/db/MongoStore.java

```diff
@@ -749,4 +749,23 @@ public boolean setMp4Muxing(String streamId, int enabled) {
 
 	}
 
+	@Override
+	public void clearStreamsOnThisServer() {
+		String ip = DBUtils.getHostAddress();
+		Query<Broadcast> query = datastore.createQuery(Broadcast.class);
+		query.or(query.criteria("originAdress").doesNotExist(), //check for non cluster mode
+				query.criteria("originAdress").equal(ip));
+		if(query.count() > 0) {
+			logger.error("There are {} streams for {} at start. They are deleted now.", query.count(), ip);
+			datastore.delete(query);
```

Comment:
It will be good to check if delete operation is successfull and log about it if fails.